### PR TITLE
feat(WebTorrent): add missing types

### PR DESCRIPTION
**Pull request by prostarz**
- Added missing `throttleDownload` and `throttleUpload`
https://webtorrent.io/docs

Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:

- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://webtorrent.io/docs
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/webtorrent/index.d.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e2f4a1d520d72730dd8053597dfeb152abd299e5
</p>

File: types/webtorrent/index.d.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ declare namespace WebTorrent {
         (config?: Options): Instance;
         WEBRTC_SUPPORT: boolean;
     }
-
+
```
Comment (RyanCavanaugh) [inline types/webtorrent/index.d.ts:16]: Remove extraneous whitespace here and on line 132

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

File: types/webtorrent/index.d.ts (hunk 1 above)
Comment (RyanCavanaugh) [reply types/webtorrent/index.d.ts]: and on line 132

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by prostarz** [comment]
Ready to merge

**Pull request merged**